src/test/java/org/gridsuite/study/server/SingleLineDiagramTest.java

```diff
@@ -93,6 +93,8 @@ class SingleLineDiagramTest {
     private static final String CASE_UUID_STRING = "00000000-8cf0-11bd-b23e-10b96e4ef00d";
     private static final UUID CASE_UUID = UUID.fromString(CASE_UUID_STRING);
     private static final UUID LOADFLOW_PARAMETERS_UUID = UUID.fromString("0c0f1efd-bd22-4a75-83d3-9e530245c7f4");
+    private static final String NAD_CONFIG_UUID = "fcb7a608-9a98-4244-aa59-e8ee8226ada5";
+    private static final String UNKNOWN_NAD_CONFIG_UUID = "71f2ad5d-fb28-4d5a-b13e-52cc6edb8cb6";
```

Comment:
```suggestion
    private static final String NAD_CONFIG_UUID_STRING = UUID.randomUUID().toString();
    private static final String UNKNOWN_NAD_CONFIG_UUID_STRING = UUID.randomUUID().toString();
```

Reply:
I moved my tests to a new file and only use Wiremock